src/templates/ucp.html

```diff
@@ -145,22 +146,32 @@ <h3>{{ 'PDF configuration'|trans }}</h3>
         <br>
 
         <input id='cjk_fonts' type='checkbox' name='cjk_fonts'
-          {{ App.Users.userData.cjk_fonts == '1' ? " checked='checked'" }}
+          {{ App.Users.userData.cjk_fonts == '1' ? " checked='checked'" }}{# <!-- lgtm[js/duplicate-html-attribute] -->
+          #}
         />
         <label for='cjk_fonts'>{{ 'Enable Chinese, Japanese and Korean fonts in PDF generation (WARNING: this will dramatically increase the file size unless you disable PDF/A)'|trans }}</label>
         <br>
         <input id='pdfa' type='checkbox' name='pdfa'
-          {{ App.Users.userData.pdfa == '1' ? " checked='checked'" }}
+          {{ App.Users.userData.pdfa == '1' ? " checked='checked'" }}{# <!-- lgtm[js/duplicate-html-attribute] -->
+          #}
         />
         <label for='pdfa'>{{ 'Generate PDF/A compliant pdfs (will embed the fonts and make the file bigger)'|trans }}</label>
         <br>
+        <input id='pdf_tex' type='checkbox' name='pdf_tex'
+          {{ App.Users.userData.pdf_tex == '1' ? " checked='checked'" }}{# <!-- lgtm[js/duplicate-html-attribute] -->
+          #}
+        />
+        <label for='pdf_tex'>{{ 'Typeset formulas written in TeX/LaTeX. WARNING: Reaction arrows with optional arguments will not be typeset correctly'|trans }}, <a href='https://mhchem.github.io/MathJax-mhchem/' target='_blank'>mhchem for MathJax manual</a></label>
```

Comment:
I don't think we should add yet another user preference. Tex should be rendered by default, I don't see a case where a user would be happy with seeing the raw tex instead of beautiful equations. It also removes a lot of noise from this pr.